portfolio/src/main/webapp/script.js

```diff
@@ -17,12 +17,35 @@
  */
 function addRandomGreeting() {
   const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+      ['Hello!', '¡Hola!', 'Ni Hao!', 'Bonjour!', 'Namaste!', 
+      'Sawasdee Kha!', 'Konnichiwa!'];
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
+
+  const curr = greetingContainer.innerText
+  if (curr === greeting) {
+    greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  }
+
   greetingContainer.innerText = greeting;
 }
+
+function addRandomQuote() {
+  const quotes = ['We were on a break!', "Joey doesn't share food", 
```

Comment:
for a moment i thought you were hitting an API to get a quote :) now i see it's hardcoded *sadface*.

Reply:
Ya :( Maybe I'll change it later if I find an API for it